### PR TITLE
Update headers and code style

### PR DESCRIPTION
**Pull request by standage**
In the most recent PR, I contributed a bit of code which is stylistically different from the rest of the code base. I wanted to follow up in a dedicated branch and make things more consistent. But rather than changing my new code into line with the rest of lusSTR, I figured now is a good a time as any to bring lusSTR into line with the majority of our other ongoing projects! :smiley:

Since we started working on lusSTR, many of our other projects have moved beyond style *checks* and adopted the strategy of completely delegating style concerns to a code autoformatting program, `black` (as in, any color you like as long as it's black). In this PR I've updated the Makefile to support both style *checks* and code formatting. As long as there's a git pre-commit hook (see the final two lines of "make devenv"), then git will check the code style when you commit and abort the commit if there's a style issue, preventing those frustrating CI build failures due to style violations. If a commit fails due to code style, you should be able to run `make format` to fix it, `git diff` to see what the autoformatter changed (if you have *any* interest), `git add`, and then `git commit` again.

I've also updated the copyright and license language in the code headers because...reasons.

Trying to make sense of the code diffs for this PR is unreasonable and futile—rest assured that no changes to the code were made aside from stylistic/aesthetic changes. The test suite still passes.

(I hope you don't have any other branches waiting in the wings. If you do, I'd be happy to put this PR on the back burner and revisit at a better time.)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,13 +8,17 @@ help: Makefile
...
 
-## style:     check code style against PEP8
+## style:     check code style
 style:
-	pycodestyle --max-line-length=99 lusSTR/*.py lusSTR/tests/test_*.py
+	black --line-length=99 --check *.py lusSTR/*.py lusSTR/tests/test_*.py
+
+## format:    auto-reformat code with Black
+format:
+	black --line-length=99 *.py lusSTR/*.py lusSTR/tests/test_*.py
```
Comment (standage) [inline Makefile:17]: This is the main event.

File: Makefile [hunk 2]
```diff
@@ -8,13 +8,17 @@ help: Makefile
...
 
 ## devenv:    configure a development environment
 devenv:
-	conda install pycodestyle pytest pytest-cov
+	conda install black==22.6 pytest pytest-cov
```
Comment (standage) [inline Makefile:21]: With supporting changes here

File: .github/workflows/cibuild.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install
       run: |
-        python -m pip install --upgrade pip pytest pytest-cov pycodestyle
+        python -m pip install --upgrade pip pytest pytest-cov black==22.6
```
Comment (standage) [inline .github/workflows/cibuild.yml:22]: and here.

File: lusSTR/filter_settings.py [hunk 4]
```diff
@@ -98,181 +102,188 @@ def thresholds(filter, metadata, locus_reads, quest_al_reads):
...
             all_type, stut_perc = minus2_stutter(
-                all_type, stutter_thresh_reads, forward_thresh, stutter_thresh_reads,
-                ref_reads, al1_ref_reads, quest_al_reads
+                all_type,
+                stutter_thresh_reads,
+                forward_thresh,
+                stutter_thresh_reads,
+                ref_reads,
+                al1_ref_reads,
+                quest_al_reads,
```
Comment (standage) [inline lusSTR/filter_settings.py:271]: And then about seventy bajillion changes related to quotation marks and long parameter lists.

**Comment by rnmitchell** [comment]
Ok awesome! This looks great to me. Thanks for doing that!

**Review by rnmitchell — APPROVED** [review]

**Pull request merged**